app/views/shared/_bottom_cta.html.erb

```diff
@@ -2,6 +2,7 @@
   <p class="bottom-cta__heading"><%= heading %></p>
 
   <p class="bottom-cta__sub-heading"><%= subheading %></p>
-
-  <%= button %>
+  <% if current_user.nil? %>
```

Comment:
This might cause issues. We're currently rendering a CTA in the FAQ page which links to the Gitter chatrooms. With your condition in place, the gitter link wouldn't show up in case a user's signed in.

https://github.com/TheOdinProject/theodinproject/blob/redesign/app/views/static_pages/faq.html.erb#L11